interface/src/avatar/MyAvatar.h

```diff
@@ -1598,6 +1598,14 @@ class MyAvatar : public Avatar {
      */
     Q_INVOKABLE float getAvatarScale();
 
+    /**jsdoc
+     * Gets the scale for the avatar that makes the avatar's eye height match the user's real-world eye height (derived from
+     * the 'User real world height' setting).
```

Comment:
```suggestion
     * the "User real world height" Interface setting).
```
Vircadia's JSDoc tends to use double quotes for quoting text.

Reply:
Somehow got a duplicate suggestion here for MyAvatar.h line 1603.  I committed the change, so I'll mark this as resolved :)